app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\AdminLoginRequest;
+use App\Models\Admin;
+use App\Traits\LoginTrait;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Route;
+use Laravel\Passport\Client;
+
+class LoginController extends Controller
+{
+    use LoginTrait;
+
+    function adminRegister(Request $request)
+    {   
+    /**
+     * Get a validator for an incoming registration request.
+     *
+     * @param  array  $request
+     * @return \Illuminate\Contracts\Validation\Validator
+     */
+    $valid = validator($request->only('email', 'name', 'password','phone'), [
+        'name' => 'required|string|max:100',
+        'email' => 'required|string|email|max:255|unique:admins,email',
+        'password' => 'required|min:6',
+        'phone' => 'required|unique:admins,phone',
+    ]);
+//is_super
+    if ($valid->fails()) {
+        // $jsonError=response()->json($valid->errors()->all(), 400);
+        // return Response::json($jsonError);
+        return response()->json($valid->errors()->all(), 422);
+    }
+
+    $data = request()->only('email','name','password','phone');
+
+    $user = Admin::create([
+        'name' => $data['name'],
+        'email' => $data['email'],
+        'password' => bcrypt($data['password']),
+        'phone' => $data['phone'],
+        'is_super' => 0,
+    ]);
```

Comment:
Use service and repo to organize your code instead of writing everything in the controller